.changes/unreleased/tinymce-TINY-11001-2024-06-05.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Changed
+body: Replaced tiny branding logo with `Build with TinyMCE` text and logo.
```

Comment:
Does this need to be documented? It's more of a maintenance task than a change and won't have a release notes entry

Reply:
One was added when changing from the text from v5 to v6 🤪 Happy to remove if no objections.

Reply:
Yes this needs to be logged since it does these things:
* Changes the CSS that could break custom skins
* Changes the I18n that could break custom language packs.
* The logo and text is wider than before so might cause rendering issues on smaller editor instances.